src/System.Security.Cryptography.X509Certificates/tests/X509StoreTests.cs

```diff
@@ -88,6 +97,32 @@ public static void Constructor_StoreHandle_Unix()
             Assert.Throws<PlatformNotSupportedException>(() => new X509Chain(IntPtr.Zero));
         }
 
+        [Fact]
+        public static void Constructor_OpenFlags()
+        {
+            using (X509Store store = new X509Store(StoreName.My, StoreLocation.CurrentUser, OpenFlags.ReadOnly))
+            {
+                Assert.True(store.IsOpen);
+            }
+        }
+
+        [Fact]
+        public static void Constructor_OpenFlags_StoreName()
+        {
+            using (X509Store store = new X509Store("My", StoreLocation.CurrentUser, OpenFlags.ReadOnly))
+            {
+                Assert.True(store.IsOpen);
```

Comment:
Please add some more tests:

* Using the current ctors and calling Open directly, is IsOpen correct.
* Call Close or Dispose on an object still in scope, does IsOpen go to false?
* Call Open again after closing it, is IsOpen true again?
* Use the new ctor, then call Open again, does IsOpen stay open?